hs_explore/management/commands/fill_go_stop_words.py

```diff
@@ -0,0 +1,125 @@
+from hs_explore.models import LDAWord
+from django.core.management.base import BaseCommand
+from hs_odm2.models import ODM2Variable
+
+
+def split_csdms():
```

Comment:
block comment """what this does""", etc.  Prune down management commands to eliminate test commands and concentrate on production commands needed for normal running.

Reply:
Anything that modifies the database should ideally be a custom migration rather than a management command. This way, no one has to worry about it.